### ISSUE TITLE
Disable 'height' on ScrollableView

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#699]
@childbs, @jhpedemonte, @billreed63, @peller

We need to investigate what's going on with ScrollableView and the 'height' property. It appears that the JS logic for ScrollableView is listening to resize events and updating element.style on ScrollableView. What this means is that if a user attempts to change 'height' via element.style for ScrollableView, their value never gets assigned.

One possible solution to this is to disable the 'height' property in the Properties palette for ScrollableView because (so it looks at the moment) users aren't supposed to change that property. We have the ability to disable element.style for a particular property on a particular widget via theme metadata (e.g., we disable 'width' via element.style on dijit.form.Button). But it maybe quite a lot to bit off building theme metadata for the dojox.mobile widgets.

SwapView probably has same issue.

**Comment by JonFerraiolo** [comment]
@peller, @billreed63 - ok for one of you to grab this bug.

**Comment by jhpedemonte** [comment]
First, both **ScrollableView** and **SwapView** (and **View**?) should have `"resizable": "none"` set in **widgets.json**.

Then, maybe we can update the Properties View code to check the `resizable` property and disable **Properties->Layout->width/height** as needed. (The `resizable` prop is defined [here](https://github.com/maqetta/maqetta/wiki/widgets.json-spec).)

**Comment by JonFerraiolo** [comment]
I think views need to be resizable in both directions, but particularly height. It is common that a UI will have a TabBar at the top and IconContainer at the bottom, with swappable views in the middle. For those scenarios, Views shouldn't have height:100%

**Comment by jhpedemonte** [comment]
Yes to the 2nd sentence.  Dojox Mobile takes care of all this for the user -- no resizing necessary.  As long as you lay out the UI properly and use the correct attributes, then the Dojox Mobile code sizes the views correctly.

For example, take a look at the HTML here: http://stackoverflow.com/questions/7154398/layouting-dojo-mobile-app/7166302.  There are 3 sibling views, each with a Heading and an app-leve TabBar.  The first view has 3 child Views and an extra TabBar.  Everything just works and the Views are properly sized, even though there are no explicit `width` or `height` attrs in the file.

**Comment by JonFerraiolo** [comment]
Pushing this to M4. Things are OK for time being as is with height:100%.

**Comment by JonFerraiolo** [comment]
Pushing to M5

**Comment by JonFerraiolo** [comment]
Reassigning to @jhpedemonte who can feel free to push to M6

**Comment by jhpedemonte** [comment]
-> M6

**Comment by doronrosenberg** [comment]
-> M7
